packages/e2e/pubspec.yaml

```diff
@@ -1,6 +1,6 @@
 name: e2e
 description: Runs tests that use the flutter_test API as integration tests.
-version: 0.3.0+1
+version: 0.3.1
```

Comment:
Because this is a breaking change, perhaps is warrants a bump to `0.4.0`?

Reply:
done.

Reply:
It seems that you haven't pushed this yet.

Reply:
no I haven't.  I pushed it now. I was also addressing the other comments :)